src/configurationProvider.ts

```diff
@@ -153,14 +151,26 @@ export class CSharpConfigurationProvider implements vscode.DebugConfigurationPro
 	 */
     resolveDebugConfiguration(folder: vscode.WorkspaceFolder | undefined, config: vscode.DebugConfiguration, token?: vscode.CancellationToken): vscode.ProviderResult<vscode.DebugConfiguration> {
 
-        if (!config.type)
-        {
+        if (!config.type) {
             // If the config doesn't look functional force VSCode to open a configuration file https://github.com/Microsoft/vscode/issues/54213
             return null;
         }
 
-        if (config.request === "launch")
-        {
+        if (config.request === "attach" && config.processCommand) {
+            const processCommand = config.processCommand.replace(/\${workspaceFolder}/g, folder.uri.fsPath);
+            delete config.processCommand;
+            return this.attachItemsProvider.getAttachItems()
```

Comment:
>return [](start = 12, length = 6)

nit: make this an async method and use `await`.

Reply:
Done